README.md

```diff
@@ -1,2 +1,8 @@
-# ariadne
-A functional implementation of a Python GraphQL server
+# Ariadne: Build functional GraphQL APIs with just enough amount of magic.
```

Comment:
Suggestion: drop "amount of" and the trailing dot

Reply:
Maybe: "Build functional GraphQL APIs in Python [...]"? If we want to build hype around it in the open-source community we should make it clear and obvious from the first sentence.